benchmarks/device_uvector/device_uvector_bench.cu

```diff
@@ -30,6 +29,9 @@
 
 #include <cuda_runtime_api.h>
 
+#include <thrust/device_vector.h>
+#include <thrust/memory.h>
```

Comment:
I can't quite tell which header is supposed to contain the public API of `thrust::raw_pointer_cast`. It doesn't seem to be a detail function. I settled on `memory.h` but am open to alternative suggestions.